sys/shell/shell.c

```diff
@@ -227,42 +236,75 @@ static void handle_input_line(const shell_command_t *command_list, char *line)
     }
 }
 
+/**
+ * Read a single line from standard input into a buffer.
```

Comment:
prepend `@brief` to have proper doxygen style

Reply:
Fixed, thanks ;)